BCStudentSoftwareDevTeam/celts
# Delete event issue 43

This PR fixes issue #43 

Summary: 
We created the ability for an admin to delete an Event after creation. We merged the Edit events page from a separate branch and simply added a Cancel event button that will hide if you are creating an event and show if you are deleting one.  We then made a `deleteEvent` function that will find the event by the given Id and recursively delete every instance of the event ID we set delete nullable to true so the records will be thoroughly deleted.

To Test: 
You can run the test suite and see our function will pass. You can also visit this URL: `http://<YOURIP>/1/2/edit_event` and you will see a red `Cancel Event` button upon clicking you will get a message notifying you that the event has been successfully canceled. You can then check the database in the events table and see it is no longer there. You can also visit the URL: `http://<YOURIP>/1/create_event` and see that the Cancel Event button does not show.

Notes: This page depends on the Edit Events page. Any changes to that page will require this PR to be updated.
